src/javatests/com/google/cloud/bigquery/dwhassessment/hooks/logger/EventRecordConstructorTest.java

```diff
@@ -332,7 +385,9 @@ public static Builder builder() {
       return new AutoValue_EventRecordConstructorTest_CountersGroupHolder.Builder();
     }
 
-    /** Builder for {@link CountersGroupHolder} */
+    /**
```

Comment:
Formatting changes are unexpected, google-java-format allows one-line javadoc comments and I also rerun formatting on all files few PRs ago. Could you verify google-java-format is enabled for all files in Intelij and rerun formatting?